src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -331,16 +331,16 @@ protected function get_views() {
 				number_format_i18n( $this->user_posts_count )
 			);
 
-			$mine = $this->get_edit_link( $mine_args, $mine_inner_html, $class );
```

Comment:
FYI: `WP_Posts_List_Table::get_edit_link()` is still used by:
- `WP_Posts_List_Table::column_author()`
- `WP_Posts_List_Table::column_default()`

These can be replaced to use `get_views_links()`, but this would lose context.

This is another use case for a general function like the proposed [wp_create_links()](https://github.com/WordPress/wordpress-develop/pull/2380).